.github/workflows/my-ci-pipeline.yml

```
@@ -0,0 +1,20 @@
+name: Continuous Integration
+on: 
+  push:
+    paths-ignore: 
+      - README.md
+  pull_request:
+    paths-ignore: 
+      - README.md
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

```suggestion
    - uses: actions/checkout@v4
```
I know this is based on our default recommendation, but worth noting that [this action is actually on v4](https://github.com/actions/checkout) which will clear up some warnings 🙂